staging/src/k8s.io/apimachinery/pkg/types/patch.go

```diff
@@ -25,4 +25,5 @@ const (
 	JSONPatchType           PatchType = "application/json-patch+json"
 	MergePatchType          PatchType = "application/merge-patch+json"
 	StrategicMergePatchType PatchType = "application/strategic-merge-patch+json"
+	ApplyPatchType          PatchType = "application/apply-patch+yaml"
```

Comment:
Will `application/apply-patch+json` be supported too?

Reply:
Good question, we've decided not to do that because json is valid yaml. Feel free to send json directly to this end-point.

Reply:
Adding a comment there might be helpful.